go/vt/vthash/highway/highwayhash.go

```diff
@@ -142,6 +142,12 @@ func (d *Digest) Write(p []byte) (n int, err error) {
 	return
 }
 
+func (d *Digest) WriteUint16(x uint16) (int, error) {
+	var b [2]byte
+	binary.LittleEndian.PutUint16(b[:], x)
```

Comment:
This can use https://pkg.go.dev/encoding/binary#Write, or is that much slower here?

Reply:
binary.Write uses reflection, this is simple and efficient in those terms. Do not see a reason to add a benchmark to validate it.